src/privateStore.js

```diff
@@ -46,11 +46,19 @@ class PrivateStore extends KeyValueStore {
   _encryptEntry (entry) {
     if (typeof entry === 'undefined') throw new Error('Entry to encrypt cannot be undefined')
 
-    return this.muportDID.symEncrypt(JSON.stringify(entry))
+    return this.muportDID.symEncrypt(this._pad(JSON.stringify(entry)))
   }
 
   _decryptEntry ({ ciphertext, nonce }) {
-    return JSON.parse(this.muportDID.symDecrypt(ciphertext, nonce))
+    return JSON.parse(this._unpad(this.muportDID.symDecrypt(ciphertext, nonce)))
+  }
+
+  _pad (val, blockSize = 24) {
+    return `${val}${'\0'.repeat(blockSize - (val.length % blockSize))}`
```

Comment:
@zachferland If the value to be padded is already a multiple of `blockSize` we still add another `blockSize` number of zeros, is that intended?

Reply:
nope, good point, no need to pad in that case, I can adjust that